### PR TITLE
Edit & Add Sayings

### PR DESCRIPTION
**Pull request by jsmorgan42**
Changes included:
- Added view controller that contains keyboard layout & modified top bar layout
- Implemented editing the core data object when the user confirms an edit
- Implemented add sayings screen

Screen Shot 2020-03-18 at 10 36 14 AM

Screen Shot 2020-03-18 at 10 35 39 AM

Screen Shot 2020-03-18 at 10 35 32 AM

### CODE REVIEW

File: Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift [hunk 1]
```diff
@@ -42,5 +42,11 @@ class EditSayingsViewController: UIViewController {
     @IBAction func backToSettings(_ sender: Any) {
         self.navigationController?.popViewController(animated: true)
     }
-
+    
+    @IBAction func addPhrasePressed(_ sender: Any) {
+        if let vc = self.storyboard?.instantiateViewController(identifier: "EditSaying") as? EditKeyboardViewController {
+            show(vc, sender: nil)
+        }
+
```
Comment (Clstroud) [inline Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift]:
```suggestion
```

**Review by Clstroud — APPROVED** [review]
👍

File: Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift (hunk 1 above)
Comment (jsmorgan42) [reply Vocable/Features/Settings/EditSayings/EditSayingsViewController.swift]: you could've just said "whitespace" like everyone else

**Review by stevefosterwta — APPROVED** [review]
👍

**Pull request merged**